webapp/src/dogma/common/components/DataTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { Table, Thead, Tbody, Tr, Th, Td, chakra, Text } from '@chakra-ui/react';
+import { TriangleDownIcon, TriangleUpIcon } from '@chakra-ui/icons';
+import {
+  useReactTable,
+  flexRender,
+  getCoreRowModel,
+  ColumnDef,
+  SortingState,
+  getSortedRowModel,
+  ColumnFiltersState,
+  getFilteredRowModel,
+} from '@tanstack/react-table';
+import { Filter } from './Filter';
```

Comment:
Should we use a relative path from the root?
Could you apply this eslint plugin?
https://www.npmjs.com/package/eslint-plugin-no-relative-import-paths

Reply:
I couldn't get the plugin to work but I've fixed the relative imports.

Reply:
Finally found it. For VS Code, changing the following setting will set the auto import to non-relative. 
<img width="606" alt="Screen Shot 2022-11-30 at 23 16 36" src="https://user-images.githubusercontent.com/5079588/204850985-88d74609-5705-4617-b8d9-f47b2ec40fcd.png">